vscode/vscode.md

```diff
@@ -0,0 +1,93 @@
+This section outlines how to set up the application in the Visual Studio Code.
+
+## Prerequisites
+
+* Java SE 8, Java SE 11 or Java SE 17 is required.
+* Payara Server 5 is required. You can download Payara Server 5 from [here](https://www.payara.fish/downloads/).
+* The Visual Studio Code is required. You can download the Visual Studio Code from [here](https://code.visualstudio.com/download).
+* [Java Extension Pack, by Microsoft](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack)
+
+## Download
+
+[Download](https://github.com/eclipse-ee4j/cargotracker/archive/master.zip) the source code zip file and expand it somewhere in your file system. Note that this is a Maven project.
+
+## Visual Studio Code Setup
+
+You need to install the payara extensions from the Visual Studio Code marketplace :
+
+* [Payara Tools, by Payara](https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode)
+
+You can install extensions from Extensions view (Ctrl + Shift + X) in the activity bar of Visual Studio Code:

```

Comment:
This needs to describe the steps in far greater detail. Please use the Eclipse IDE instructions as a starting point to understand what would be sufficient for project documentation.

Reply:
* [Payara Tools, by Payara](https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode)

Go to extension -> Visual studio code marketplace. Search for "Payara" and install Payara Tools.

You can install extensions from the Extensions view (Ctrl + Shift + X) in the activity bar of Visual Studio Code:

Do I need to describe more?
At Line no. 38 I described  about payara installation in detail.

Reply:
Without doing this myself, it’s hard to tell. The user needs clear, step-by-step instructions on what they should be doing. They should not need to rely on looking at screenshots or need to guess what to do. Again, please look at the existing Eclipse IDE instructions for an example.